CMakeLists.txt

```diff
@@ -99,39 +99,38 @@ if( EXIV2_BUILD_FUZZ_TESTS )
     add_subdirectory ( fuzz )
 endif()
 
-if( EXIV2_BUILD_SAMPLES )
-    add_subdirectory( samples )
-    get_directory_property(SAMPLES DIRECTORY samples DEFINITION APPLICATIONS)
-
-    if (Python3_Interpreter_FOUND)
-        add_test(NAME bashTests
-            WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
-            COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose bash_tests
-        )
-    endif()
-endif()
+if(EXIV2_BUILD_EXIV2_COMMAND)
```

Comment:
I also noticed that before we were adding the python tests to CTEST even when `EXIV2_BUILD_EXIV2_COMMAND` was OFF 😅